plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/expression/RewriteVarcharConstant.java

```diff
@@ -40,6 +40,9 @@ public Pattern<Constant> getPattern()
     public Optional<String> rewrite(Constant constant, Captures captures, RewriteContext<String> context)
     {
         Slice slice = (Slice) constant.getValue();
+        if (slice == null) {
```

Comment:
same in `RewriteExactNumericConstant`

Reply:
Can you please move the changes here and in RewriteExactNumericConstant to a separate PR?

Would be good to fix this before this code gets copied to other places.